govcms_ckan.module

```diff
@@ -11,6 +11,14 @@
 define('GOVCMS_CKAN_CONFIG_PATH', 'admin/config/services/govcms-ckan');
 
 /**
+ * Limit of records retrieved via the API. CKAN default is 100.
+ *
+ * TODO: Move this to a configuration (somewhere). For now, we really want all
+ * records as we don't have any way to 'request more' results from the api.
+ */
+define('GOVCMS_CKAN_DATASET_RECORD_LIMIT', 1000000);
```

Comment:
Do we have any real world examples to better understand what this number should be? 1m seems unreasonably high, we don't want to have incredibly long running php processes as a result of this